app/ide-desktop/lib/client/src/config/parser.ts

```diff
@@ -0,0 +1,319 @@
+                    firstSentenceSplit == -1
+                        ? option.description
+                        : option.description.slice(0, firstSentenceSplit + 1)
+                let otherSentences = option.description.slice(firstSentence.length)
+
+                const def = option.defaultDescription ?? option.default
+                let defaults = ''
+                if (def != null && def !== '') {
+                    defaults = ` Defaults to ${chalk.green(def)}.`
+                }
+                let description = firstSentence + defaults + chalk.gray(otherSentences)
+                const lines = wordWrap(description, rightWidth).map(
+                    line => line + ' '.repeat(rightWidth - stringLength(line))
+                )
+                const right = lines.join('\n' + ' '.repeat(leftWidth))
+                console.log(left + right)
+            }
+        }
+    }
+}
+
+function wordWrap(str: string, width: number): string[] {
+    if (width <= 0) {
+        return []
+    }
+    let firstLine = true
+    let line = ''
+    const lines = []
+    const inputLines = str.split('\n')
+    for (const inputLine of inputLines) {
+        if (!firstLine) {
+            lines.push(line)
+            line = ''
+        }
+        firstLine = false
+        for (let word of inputLine.split(' ')) {
+            if (stringLength(word) > width) {
+                if (line.length > 0) {
+                    lines.push(line)
+                    line = ''
+                }
+                const wordChunks = []
+                while (stringLength(word) > width) {
+                    wordChunks.push(word.slice(0, width))
+                    word = word.slice(width)
+                }
+                wordChunks.push(word)
+                for (const wordChunk of wordChunks) {
+                    lines.push(wordChunk)
+                }
+            } else {
+                if (stringLength(line) + stringLength(word) >= width) {
+                    lines.push(line)
+                    line = ''
+                }
+                if (line.length != 0) {
+                    line += ' '
+                }
+                line += word
+            }
+        }
+    }
+    if (line) {
+        lines.push(line)
+    }
+    return lines
+}
+
+// ======================
+// === Chrome Options ===
+// ======================
+
+export class ChromeOption {
+    constructor(public name: string, public value?: undefined | string) {}
+
+    display(): string {
+        const value = this.value == null ? '' : `=${this.value}`
+        return `--${this.name}${value}`
+    }
+}
+
+/** Replaces `-no-...` with `--no-...`. This is a hotfix for Yargs bug:
+ * https://github.com/yargs/yargs-parser/issues/468 */
+function fixArgvNoPrefix(argv: string[]): string[] {
+    const singleDashPrefix = '-no-'
+    const doubleDashPrefix = '--no-'
+    return argv.map(arg => {
+        if (arg.startsWith(singleDashPrefix)) {
```

Comment:
Shouldn't this be done only for boolean arguments?

Reply:
No, we want to catch a bug when someone uses this option for non boolean arguments. Now, it will print the parsing error correctly.